spring-boot-admin-server/src/main/java/de/codecentric/boot/admin/server/config/AdminServerProperties.java

```diff
@@ -127,6 +129,56 @@ public String path(String path) {
 
 	}
 
+	@lombok.Data
+	public static class InstanceAuthProperties {
+
+		/**
+		 * Whether or not to use configuration properties as a source for instance
+		 * credentials <br/>
+		 * Default: false
+		 */
+		private boolean enabled = false;
```

Comment:
If we don't provide any default passwords, I'd suggest to enable the feature by default.

Reply:
now enabled by default, and updated the docs to reflect that.